### DISCUSSION TITLE
Entry level projects

### DISCUSSION THREAD

**Discussion by akhilsamvarghese** [community/community#135420] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

# confused on what to work on

**Comment by AghilesTMA** [comment, marked as answer]
well if you are a web developer you can work on a simple Ecomerce website, task management App, or even a chat application, here are youtube tutorials about the apps i mentioned above:
-Ecommerce: https://youtu.be/I0BOUiFe9WY?si=6v5hp_LlpJ52slr4
-task manager: https://youtu.be/kPKXmEp_wfU?si=o3-ICQuziGcDYo0j
-Chat app: https://youtu.be/HwCqsOis894?si=GkejxSclayx546uO

**Reply by akhilsamvarghese** [reply]
Thanks mate 😌
